arcgis-ios-sdk-samples/Display information/Show legend/README.md

```diff
@@ -1,20 +1,31 @@
 # Show legend
 
-This sample demonstrates how to show a legend for all the operational layers in the map
+Show a legend for all the operational layers in the map.
 
-## How to use the sample
+![Image of Show legend 1](show-legend-1.png)
+![Image of Show legend 2](show-legend-2.png)
+![Image of Show legend 3](show-legend-3.png)
 
-The map in the app comes loaded with multiple operational layers. When you tap on the `Legend` button in the bottom toolbar you are shown a list of legends for each of those layers and their sublayers.
+## Use case
 
-![](image1.png)
-![](image2.png)
-![](image3.png)
+Legends are used to describe what each symbol on a map represents. A common format is to show an image of the symbol alongside of a text description of what that symbol represents. This sample demonstrates how to show a legend for all the operational layers in the map.
 
-## How it works
+## How to use the sample
 
-The sample runs through each layer in the operational layers list and calls `fetchLegendInfos(completion:)` method on it or its sublayers. On completion, the method returns an array of `AGSLegendInfo` objects. Each of these objects contain the symbol and the name describing the legend. 
+1. Open the sample
+2. Scroll through the legends to see the various elements that represent features on the map.
+
+## How it works
 
+1. Layers implement the `AGSLayerContent` interface, which contain a list of `AGSLegendInfo`. `AGSLegendInfo` contains an `AGSSymbol` and a name string. `AGSLayerContent.fetchLegendInfos(completion:)` must be called on each `AGSLayerContent` instance to fetch the info from the data.
+2. For each symbol in the `AGSLegendInfo` list, `AGSSymbol.createSwatch(completion:)` must be called so that an image of the `AGSSymbol` is returned.
```

Comment:
```suggestion
2. For each symbol in the `AGSLegendInfo` array, `AGSSymbol.createSwatch(completion:)` must be called so that an image of the `AGSSymbol` is returned.
```